researchstudio-sat/webofneeds
# Adds post-context-dropdown to post-visitor-view

Removes unused _need-tab-bar.scss file

fixes #2531 

HowToTest:
- Open a (shared) post-uri
- see that the post-context-dropdown is available in the visitor-title-bar
- click report post
- see that the email report post opens
